src/main/java/net/mcreator/ui/wysiwyg/WYSIWYGEditor.java

```diff
@@ -369,6 +371,22 @@ public WYSIWYGEditor(final MCreator mcreator, boolean isNotOverlayType) {
 				.join(FlowLayout.LEFT, L10N.label("elementgui.gui.width_height"), spa1, new JLabel("x"), spa2));
 		adds2.add(PanelUtils.join(FlowLayout.LEFT, L10N.label("elementgui.gui.inventory_offset"), invOffX, invOffY));
 		adds2.add(PanelUtils.join(FlowLayout.LEFT, renderBgLayer));
+
+		customBg.addActionListener(e -> editor.repaint());
+		customBg.setRenderer(new WTextureComboBoxRenderer.OtherTextures(mcreator.getWorkspace()));
+		customBg.setPrototypeDisplayValue("XXXXXXXXXXX");
+		ComponentUtils.deriveFont(customBg, 16);
+		JButton importbgtexture = new JButton(UIRES.get("18px.add"));
+		importbgtexture.setToolTipText(L10N.t("elementgui.gui.background_texture_import"));
+		importbgtexture.setOpaque(false);
+		importbgtexture.addActionListener(e -> {

```

Comment:
I suggest either leaving this button out or add it to the overlay editor too to keep things consistent.